docs/running-rules.md

```diff
@@ -9,49 +9,49 @@ import MoreHelp from "/src/components/MoreHelp"
 
-You can run all rules on your local code or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI.
+You can run all rules on your code completely locally or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI. For more information, see [Semgrep CI overview](semgrep-ci/overview.md).
 
-The following diagram displays the current number of Semgrep rules:
-<div className="lang-container" style={{marginBottom: '20px'}}>
-  <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
-</div>
+## Running Semgrep Registry rules locally
 
-## Running Semgrep Registry rules
+You can test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code with the following command (substitute the `PATH/TO/SRC` with path to your source code):
+<pre class="language-bash"><code>semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
 
-This section explains how to run [Semgrep Registry](https://semgrep.dev/explore) rules locally. See the following examples of how you may test Semgrep rules:
+:::info
+* `--config auto` sends your repository's project URL to [Semgrep Registry](https://semgrep.dev/r) to find rules configured for your repository and as a key for cached rule recommendations.
+* When Semgrep Registry is used, [usage metrics](metrics.md) are collected by default.
+:::
 
-1. Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code:
-    ```sh
-    semgrep --config=auto path/to/src
-    ```
-2. Explore the [Semgrep Registry](https://semgrep.dev/explore) and choose a rule.
-3. On the page of the rule, click **Run Locally**.
-4. Copy the code for local install, and then add the path to the source code you want to check:
+Explore the Semgrep Registry by following these steps:
+
+1. See the [Semgrep Registry](https://semgrep.dev/explore) and choose a rule.
+2. On the page of the rule, click **Run Locally**.
+3. Copy the code for local install, and then add the path to the source code you want to check in your terminal:
     <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" <span className="placeholder">PATH/TO/SRC</span></code></pre>
-5. Optional: Run registry rules simultaneously with local rules:
-   <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/YML PATH/TO/SRC</span></code></pre>
+4. Optional: Run registry rules simultaneously with local rules:
+   <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/MYRULE.YAML PATH/TO/SRC</span></code></pre>
```

Comment:
I'll suggest PATH/TO/REGISTRY/SRC once more and then stop suggesting it if you prefer the other way

Reply:
oop, sorry I see you had that previous question "unresolved" I must have had my github view wrong. Sorry about that!

Reply:
I mean... a path to SRC is a path to the source code you want to have checked... the path to myrule.yaml is a path to a local YAML file with rules... so these are two different things. But it might be confusing really! So what about:

`PATH/TO/SEMGREP_RULES.YAML PATH/TO/SRC`

Or maybe I am missing something and you mean the `PATH/TO/REGISTRY/SRC` for the `RULESET-ID`? :-)

Reply:
omg, no I misunderstood. thanks for the clarification! I thought for some reason the SRC was you downloading the rules from the registry. ignore my comments!

Reply:
No worries, maybe more users will have problem with this... Would be great to one day try this with somebody who have not tried Semgrep before. @efortuna